src/molecules/IconLink/IconLink.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { ButtonHTMLAttributes, DetailedHTMLProps } from 'react';
+import { StyledComponentClass } from 'styled-components';
+import Icon, { IconName } from '../../atoms/Icon';
+import Typography from '../../atoms/Typography';
+import styled from '../../styled-components';
+import Theme from '../../Theme';
+// import { IconName } from '@fortawesome/fontawesome-svg-core';
+
+IconLink.defaultProps = { as: 'a' };
+
+export const IconTypography = styled(Typography)`
+  background: none;
+  border: none;
+  padding: 0;
+  color: ${props => props.theme.text};
+
+  :hover {
+    color: ${props => props.theme.primaryDark};
+    /* stylelint-disable-next-line max-nesting-depth */
```

Comment:
I don't think we need this comment

Reply:
Oh yeah. That was a remnant from when I was styling the link within. Good catch.